source/text.tex

```diff
@@ -2160,10 +2160,10 @@
 Stops if it encounters a character it cannot convert.
 It always leaves the \tcode{from_next} and \tcode{to_next} pointers
 pointing one beyond the last element successfully converted.
-If returns \tcode{noconv},
-\tcode{internT} and \tcode{externT} are the same type and
+If it returns \tcode{noconv},
+\tcode{internT} and \tcode{externT} are the same type, and
```

Comment:
Does it make sense to strike "`internT` and `externT` are the same type and the converted sequence is identical to the input sequence `[from, from_next)`"? Because it duplicates [[tab:locale.codecvt]'s last row](https://eel.is/c++draft/locale.codecvt.virtuals#tab:locale.codecvt.inout-row-5).

Reply:
Maybe it does.  However, this editorial change request tries to fix the English sentence structure from "ill-formed" to "readable", not to fix any other problems in the vicinity.